dashboard/app/models/pd/enrollment.rb

```diff
@@ -24,17 +25,48 @@
 class Pd::Enrollment < ActiveRecord::Base
   belongs_to :workshop, class_name: 'Pd::Workshop', foreign_key: :pd_workshop_id
   belongs_to :school_district
+  belongs_to :user
 
-  validates :name, :email, :school, :school_type, presence: true
+  validates :name, :email, presence: true
```

Comment:
These fields are what they fill out on the enrollment which by design does not require a Code Studio login. In order to be counted as an attendee they do need a Code Studio account, which is matched to the enrollment by email. So, at some point the enrollment email and user.email must match. I could add a validation on setting user to ensure this. Thoughts?

Presumably the name will match, but there is no guarantee. 
Once matched, it's possible for users to change their name or email at any time so in the future it may deviate from their original enrollment response. This is one of the reasons I want to store the actual user id.